hyper/http11/connection.py

```diff
@@ -137,6 +144,33 @@ def connect(self):
 
         return
 
+    @classmethod
```

Comment:
Sorry, I should have been clearer. I wasn't suggesting an **instance method**: I was suggesting a **function**. Don't hang it off a class at all, it doesn't need one.